Gemfile.lock

```diff
@@ -0,0 +1,91 @@
+PATH
```

Comment:
Don't think we should be checking in a Gemfile.lock to a gem...  unless guidance on that has changed or you have a specific reason it's needed?  Could have locally or as an example to share among devs maybe if any specific concerns, but I'm citing Yehuda [here](http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) for reference